src/FlashMessage/FlashManager.php

```diff
@@ -50,13 +52,23 @@ public function __construct(SessionInterface $session, array $types, array $cssC
     /**
      * Tells if class may handle $object for status class rendering.
      *
+     * @deprecated since sonata-project/twig-extensions 1.x and will be remove in 2.0. Use handlesType() instead.
+     *
+     * NEXT_MAJOR: remove this method
+     *
      * @param object|string $object     FlashManager or Sonata flash message type
      * @param string|null   $statusName Sonata flash message type
      *
      * @return bool
      */
     public function handlesObject($object, ?string $statusName = null)
     {
+        @trigger_error(sprintf(
+            'The "%s" methods is deprecated. Use "%s" instead.',
```

Comment:
```suggestion
            'The "%s()" method is deprecated since sonata-project/twig-extensions 1.x'
            .' and will be removed in version 2.0. Use "%s" instead.',
```